package.json

```diff
@@ -55,26 +56,26 @@
     "eslint": "^5.16.0",
     "faucet": "^0.0.1",
     "jsonpath": "^1.1.1",
+    "mkdirp": "^0.5.6",
+    "node-addon-api": "^5.0.0",
     "node-timeout": "0.0.4",
     "polyline": "^0.2.0",
     "request": "^2.88.2",
+    "rimraf": "^2.7.1",
     "tape": "^4.16.0",
     "turf": "^3.0.14",
     "uglify-js": "^3.17.0",
     "xmlbuilder": "^4.2.1",
-    "cheap-ruler": "^3.0.2",
-    "mkdirp": "^0.5.6",
-    "nan": "^2.17.0",
-    "node-cmake": "^2.5.1",
-    "rimraf": "^2.7.1"
+    "node-cmake": "^2.5.1"
   },
   "main": "lib/index.js",
   "binary": {
+    "napi_versions": [8],
```

Comment:
Version 8 is the latest and it is compatible with any Node version we currently support:
<img width="2158" alt="Screenshot 2022-11-12 at 11 57 08" src="https://user-images.githubusercontent.com/266271/201471225-1bf603d0-4967-4793-bd2a-e4710da16b2b.png">
https://nodejs.org/api/n-api.html#node-api-version-matrix